.gitignore

```diff
@@ -52,7 +52,9 @@ src/ontology/mondo.json
 src/ontology/mondo.obo
 src/ontology/mondo.owl
 src/ontology/reasoned.owl
-src/ontology/reports
+src/ontology/reports/*
```

Comment:
@matentzn You asked that I commit some files. I had to update the `.gitignore` for this:
```gitignore
src/ontology/reports/*
!src/ontology/reports/obsoletes.tsv
!src/ontology/reports/gard-mondo-mapped-obsoletes.tsv
```

First thing I want to mention is that I did not also commit the other input required for `reports/gard-mondo-mapped-obsoletes.tsv`, which is `tmp/mondo.sssom.tsv`; it's a necessary part of my current implementation of this. I did not want it to be confused with `mappings/mondo.sssom.tsv`.

Secondly, am I correct that you indeed also wanted me to commit `reports/gard-mondo-mapped-obsoletes.tsv`?

Third, I just want to critique the current setup of `reports/` in the `.gitignore`. It looks like the directory has been force ignored, and specific files have been git added without making exceptions for them explicitly in the `.gitignore`. I think that explicit is probably better. If you want, I can add exceptions for those in the `.gitignore` as well.

Reply:
Removed the `obsoletes.tsv` line since I'm no longer using the OAK command for this work. I can add it back if Nico wants a makefile command to just run the obsoletes table and add it to `reports/`, but I could do that as a PR directly to `master`.

Reply:
This is good for now!